src/infrastructure/jira/jira-client/testing/mocks.ts

```diff
@@ -1,6 +1,7 @@
 import { v4 as uuidv4 } from 'uuid';
 
 import { Duration } from '../../../../common/duration';
+import { MOCK_ISSUE_KEY } from '../../../figma/testing';
```

Comment:
**nitpick:** Our tests in `infrastructure/jira` depend on mocks from `infrastructure/figma/testing`. This can be an undesired dependency since these modules do not and should not not know about each other.

The issue happens because both `figma` and `jira` modules use the same `Issue` data. Therefore, in order to resolve this, consider moving domain-related constant  (e.g., `MOCK_ISSUE_KEY`, `MOCK_ISSUE_TITLE`) to `domain/testing` by analogue with `MOCK_ISSUE_ID`. As a result, both `infrastructure/jira` and `infrastructure/figma/testing` will either depend on their own mocks or mocks from `domain/testing`.